### ISSUE TITLE
comments are executed

### ISSUE THREAD

**Issue by alainravet** [bkeepers/dotenv#336]
file: .env.development:

```
# Uncomment to activate:
# DO_X=anything

```

with dot-env 2.2.2

```
bundler exec rails c
2.5.0 :001 > ENV['DO_X']
 => nil         <--------- GOOD
```

with dot-env 2.3.0
```
bundler exec rails c
2.5.0 :001 > ENV['DO_X']
 => "anything"         <--------- BAD
```

**Comment by jonmagic** [comment]
Thank you for the bug report. Looking into potential solutions now.

**Comment by jonmagic** [comment]
We just shipped version 2.4.0 which removes the `Parse multiline values` support added in 2.3.0 and fixes the parsing regressions.
